app/models/passenger.rb

```diff
@@ -0,0 +1,12 @@
+class Passenger < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+  validates :name, presence: true
+  validates :phone_num, presence: true
+
+  def total_trip_cost
+    costs = trips.collect { |trip| trip.cost }.inject { |sum, trip_cost| sum + trip_cost }
```

Comment:
Good use of enumerables and good work putting the business logic into the models!